x-pack/test/functional/services/cases/helpers.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import uuid from 'uuid';
+
+export function generateRandomCaseWithoutConnector() {
+  return {
+    title: 'random-' + uuid.v4(),
+    tags: ['test', uuid.v4()],
+    description: 'This is a description with id: ' + uuid.v4(),
```

Comment:
The id here will be different from the id in the title and form the id of the case. Is that intended? Again, I am not in favor of generating dynamic data if not needed.

Reply:
yes this is intended.